### DISCUSSION TITLE
Fork of the actors framework

### DISCUSSION THREAD

**Discussion by valyagolev** [quickwit-oss/quickwit#3447] [Show and tell]
I forked the quickwit-actors framework here: https://github.com/valyagolev/witty-actors , and published it as a crate (that's how much I like it). I vendored the needed quickwit dependencies. I hope it's alright.

It's a tiny bit of work so just putting it here so that whoever needs it, can find it.

**Comment by imotov** [comment]
It seems to be a private repo.

**Reply by valyagolev** [reply]
sorry about this, fixed
